recipe/emailpassword/api/signin.go

```diff
@@ -49,10 +51,12 @@ func SignInAPI(apiImplementation epmodels.APIInterface, options epmodels.APIOpti
 		return err
 	}
 	if result.WrongCredentialsError != nil {
+		supertokens.DebugLoggerWithCode[int(supertokens.API_RESPONSE)]("SignInAPI", "WRONG_CREDENTIALS_ERROR")
```

Comment:
1. Correct me if I'm wrong (I'm not that familiar with go) but this call signature seems awkward. I think this will be fixed while removing log levels, but wanted to make note of it.
2. This PR has log calls while the node PR removed them.

Reply:
Hey, think that you are looking at an older commit, you can check the files changed, they have the latest push